commands

```diff
@@ -180,6 +220,40 @@ case "$1" in
     check_container
     docker logs dokku-mongodb | tail -n 100
     ;;
+  mongodb:restore) 
+    verify_min_args 2 "$@"
+    check_container
+    check_db "$2"
+
+    fileOrDir="${3%/}"
+    # does file or folder exist?
+    if [[ ! -d "$fileOrDir" && ! -f "$fileOrDir" ]]; then
+      fail "$fileOrDir doesn't exist"
+    fi
+    # does the folder have the same name as the database? 
+    if [[ -d "$fileOrDir" && "$fileOrDir" != *"$2"  ]]; then
+      fail "The folder $fileOrDir must have the same name as the database $2"
```

Comment:
Good call. I will remove check_db. Shall I remove the <databasename> argument altogether?